src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java

```diff
@@ -412,6 +424,17 @@ null, new ItemStack(Material.WITHER_ROSE), null,
         player -> simplePlayerEffect(player, 2, new Pair<>(PotionEffectType.BAD_OMEN, 0))
     ).buildRegister(Cultivation.getInstance());
 
+    public static final Food BURGER = new Food(
```

Comment:
do we want to make this more specific or nah?
There are a bunch of burgers
Also im not a big burger eater but arent burgers with cheese automatically a cheeseburger?

Reply:
Fair, made into cheese burger :)